tests/topotests/bgp_rpki_topo1/r2/bgp_table_rmap_rpki_valid.json

```diff
@@ -0,0 +1 @@
+bgp_table_rpki_valid.json
```

Comment:
What is this?

Reply:
Means symbolic link (used in the route map test)

```
$ ls -l tests/topotests/bgp_rpki_topo1/r2/bgp_table_rmap_rpki_valid.json
lrwxrwxrwx 1 user users 25 Jan 11 14:17 tests/topotests/bgp_rpki_topo1/r2/bgp_table_rmap_rpki_valid.json -> bgp_table_rpki_valid.json

$ git diff HEAD^ --name-only 
tests/topotests/bgp_rpki_topo1/__init__.py
tests/topotests/bgp_rpki_topo1/r1/bgpd.conf
tests/topotests/bgp_rpki_topo1/r1/rtrd.py
tests/topotests/bgp_rpki_topo1/r1/staticd.conf
tests/topotests/bgp_rpki_topo1/r1/vrps.csv
tests/topotests/bgp_rpki_topo1/r1/zebra.conf
tests/topotests/bgp_rpki_topo1/r2/bgp_table_rmap_rpki_any.json
tests/topotests/bgp_rpki_topo1/r2/bgp_table_rmap_rpki_notfound.json
tests/topotests/bgp_rpki_topo1/r2/bgp_table_rmap_rpki_valid.json
tests/topotests/bgp_rpki_topo1/r2/bgp_table_rpki_any.json
tests/topotests/bgp_rpki_topo1/r2/bgp_table_rpki_notfound.json
tests/topotests/bgp_rpki_topo1/r2/bgp_table_rpki_valid.json
tests/topotests/bgp_rpki_topo1/r2/bgpd.conf
tests/topotests/bgp_rpki_topo1/r2/rpki_prefix_table.json
tests/topotests/bgp_rpki_topo1/r2/staticd.conf
tests/topotests/bgp_rpki_topo1/r2/zebra.conf
tests/topotests/bgp_rpki_topo1/test_bgp_rpki_topo1.py
```